apps/next-evans/pages/explore.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import type { GetStaticProps } from 'next';
+import type { DocumentWithMeta, Period } from '@/lib/types';
+import BackgroundImage from '@/components/core/BackgroundImage';
+import Dual from '@/components/core/Dual';
+import HeroImg from '@/public/images/explore-books.jpg';
+import NavBlock from '@/components/pages/explore/NavBlock';
+import UpdatedEditionsBlock from '@/components/pages/explore/UpdatedEditionsBlock';
+import { LANG } from '@/lib/env';
+import GettingStartedLinkBlock from '@/components/pages/explore/GettingStartedLinkBlock';
+import AudioBooksBlock from '@/components/pages/explore/AudioBooksBlock';
+import NewBooksBlock from '@/components/pages/explore/NewBooksBlock';
+import ExploreRegionsBlock from '@/components/pages/explore/RegionBlock';
+import TimelineBlock from '@/components/pages/explore/TimelineBlock';
+import AltSiteBlock from '@/components/pages/explore/AltSiteBlock';
+import { getAllDocuments, getNumDocuments } from '@/lib/db/friends';
+import { mostModernEdition } from '@/lib/editions';
+import SearchBlock from '@/components/pages/explore/SearchBlock';
+
+export const getStaticProps: GetStaticProps<Props> = async () => {
+  const documents = Object.values(await getAllDocuments());
+  const numBooksInAltLang = await getNumDocuments(LANG === `en` ? `es` : `en`);
+  return {
+    props: {
+      books: documents,
+      numBooks: documents.length,
+      numBooksInAltLang,
+    },
+  };
+};
+
+interface Props {
+  numBooks: number;
+  numBooksInAltLang: number;
+  books: Array<DocumentWithMeta>;
```

Comment:
does this mean we're using _every_ bit of data from `DocumentWithMeta`? isn't that sort of a grab-back catch-all type? do we need to `Pick` here to make sure we don't bloat the client-side reload props?

Reply:
or does this type represent something more like "the props we need to render a cover"?  if so, i wonder if we should rename it... 🤔